seq2seq/third_party/main/training_args.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2020 Google LLC
```

Comment:
We need to add the lines mentioning the original hugging face version and that we modified it.